cosmoz-treenode-navigator.js

```diff
@@ -15,24 +16,26 @@
 			that component is given
 			 */
 			data: {
```

Comment:
I don't think `data` should be kept but rather just expect a
```js
tree: {
  type: Cosmoz.Tree
}
```

like https://github.com/Neovici/cosmoz-treenode/blob/master/cosmoz-treenode.html#L45